### DISCUSSION TITLE
How to compile Dear ImGUI correctly using Android.mk ?

### DISCUSSION THREAD

**Discussion by VadimBoev** [ocornut/imgui#6430] [New Users Build/Link/Run issues ONLY!]
Version: 1.89.5
Branch: release

Good afternoon dear ImGui. I'm working on an Android app. Faced with a compilation problem. I use Android.mk.  
Why when using "LOCAL_CFLAGS := -g -dynamic -funwind-tables -wp thread -Wall -O2 -std=c++14 -fno-omit-frame-pointer -funwind-tables", there is no crash, but this setup does not really suit me. And when using "LOCAL_CPPFLAGS := -ws -fvisibility=hidden -pthread -Wall -fpack-struct=1 -O2 -std=c++14 -fexceptions" crashes. What to do, how to compile correctly, why is this happening?

**Comment by ocornut** [comment]
I’ve moved this to a Discussion thread as required by topic nature. Please note that provided the absence of more details normally requested by issue Guidelines, this is likely going to require more infos on your side to be answered properly.
